src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUnique.cs

```diff
@@ -77,7 +84,7 @@ protected override void ProcessRecord()
                 if (string.Equals(
                     inputString,
                     _lastObjectAsString,
-                    StringComparison.CurrentCulture))
+                    CaseInsensitive.IsPresent ? StringComparison.CurrentCultureIgnoreCase : StringComparison.CurrentCulture))
```

Comment:
Should we consider `OnType` above too?

Reply:
@iSazonov `Ontype` would only return one instance of each type so case sensitivity shouldn't matter here?